src/main/kotlin/com/terraformation/backend/accelerator/DeliverableCompleter.kt

```diff
@@ -66,15 +68,18 @@ class DeliverableCompleter(
       projectId: ProjectId,
       predicate: (() -> Boolean)? = null
   ) {
-    val moduleId = deliverableStore.fetchDeliverableModuleId(deliverableId)
-    val phase = moduleStore.fetchCohortPhase(moduleId)
+    systemUser.run {
+      val moduleId = deliverableStore.fetchDeliverableModuleId(deliverableId)
+      val phase = moduleStore.fetchCohortPhase(moduleId)
 
-    if (phase == CohortPhase.PreScreen || phase == CohortPhase.Application) {
-      if (predicate == null || predicate()) {
-        submissionStore.createSubmission(deliverableId, projectId, SubmissionStatus.Completed)
+      if (phase == CohortPhase.PreScreen || phase == CohortPhase.Application) {
+        if (predicate == null || predicate()) {
+          submissionStore.createSubmission(deliverableId, projectId, SubmissionStatus.Completed)
```

Comment:
It's likely we don't care, but just noting that this will cause the submission to show up as created by the system user instead of by whichever user completed the deliverable.